rules/k8s_audit_rules.yaml

```diff
@@ -174,6 +174,22 @@
   source: k8s_audit
   tags: [k8s]
 
+- rule: Create HostPid Pod
+  desc: Detect an attempt to start a pod using the host pid namespace.
+  condition: kevt and pod and kcreate and ka.req.pod.host_pid intersects (true)
```

Comment:
can you add an exception list ? e.g. `falco_hostpid_images` and `user_known_hostpid_images`

Reply:
Hi @Kaizhe, I have added an empty list for both rules. If you have any suggestions on how to find out non-malicious images that make use of these namespaces to work, let me know! Thank you